spec/requests/api/picture_spec.rb

```diff
@@ -63,26 +67,10 @@ def expect_result_to_include_picture_href(source_id)
   end
 
   describe 'POST /api/pictures' do
-    # one pixel png image encoded in Base64
```

Comment:
Even though this looks like gibberish, I sort of liked this because it was a technically a valid image, giving anyone looking for documentation in these specs some idea of what to expect. The shorter, valid base 64 code above may suffice though. @abellotti any thoughts?